scalafix-rules/src/main/scala/scalafix/internal/rule/OrganizeImports.scala

```diff
@@ -685,6 +694,122 @@ class OrganizeImports(config: OrganizeImportsConfig)
 
     Patch.addLeft(token, indented mkString "\n")
   }
+
+  private def prettyPrintImportGroup(group: Seq[Importer]): String = {
+    group
+      .map(i => "import " + importerSyntax(i))
+      .mkString("\n")
+  }
+
+  /**
+   * HACK: The Scalafix pretty-printer decides to add spaces after open and
+   * before close braces in imports with multiple importees, i.e., `import a.{
+   * b, c }` instead of `import a.{b, c}`. On the other hand, renames are
+   * pretty-printed without the extra spaces, e.g., `import a.{b => c}`. This
+   * behavior is not customizable and makes ordering imports by ASCII order
+   * complicated.
+   *
+   * This function removes the unwanted spaces as a workaround. In cases where
+   * users do want the inserted spaces, Scalafmt should be used after running
+   * the `OrganizeImports` rule.
+   */
+  private def importerSyntax(importer: Importer): String =
+    importer.pos match {
+      case pos: Position.Range =>
+        // Position found, implies that `importer` was directly parsed from the source code. Use
+        // the original parsed text to preserve the original source level formatting, but patch
+        // importee that have specific Scala 3 syntax.
+        val syntax = new StringBuilder(pos.text)
+        def patchSyntax(
+            t: Tree,
+            newSyntax: String,
+            stripEnclosingBraces: Boolean = false
+        ) = {
+          val start = t.pos.start - pos.start
+          syntax.replace(start, t.pos.end - pos.start, newSyntax)
+          val end = t.pos.start - pos.start + newSyntax.length
+
+          if (stripEnclosingBraces)
+            (
+              syntax.take(start).lastIndexOf('{'),
+              syntax.indexOf('}', end)
+            ) match {
+              case (from, to) if from != -1 && to != -1 =>
+                syntax.delete(end, to + 1)
+                syntax.delete(from, start)
+              case _ =>
+            }
```

Comment:
```diff
-import foo.{qux as bar}
+import foo.qux as bar
```

Still unsure if we want to force that as it goes against the format-preserving features.

If we don't strip, we end up having unexpected sorting in Scala 3 as we use pretty-printed brace-less syntax as sort key with the current implementation:
```
import foo.{qux as bar} // sorted as foo.qux
import foo.{bar, qux}
```

Reply:
I think stripping {} here is fine. It's actually much easier to just do as, so I would recommend that